etc/format-all.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+. "$(dirname "${BASH_SOURCE[0]}")/../.evergreen/init.sh"
+
+files="$(find "$LIBMONGOCRYPT_DIR/src" -type f -name '*.c' -o -name '*.h' -o -name '*.cpp' -o -name '*.hpp')"
+# shellcheck disable=SC2206
+IFS=$'\n' files=($files)
+bash "$LIBMONGOCRYPT_DIR/etc/format.sh" \
+    --style="file:$LIBMONGOCRYPT_DIR/.clang-format" \
+    -i \
+    "$@" \
+    -- "${files[@]}"
```

Comment:
```suggestion
    -- "${files[@]:?}"
```

Despite clang-format emitting an error when no files are given when `-i` is present, it still returns an exit code of 0 indicating success:
```bash
$ clang-format -i; echo $?
error: cannot use -i when reading from stdin.
0
```
Suggest asserting `$files` is not null by adding `:?`.